projects/packages/search/changelog/record meter: add basic chart structure with dummy data

```diff
@@ -1,4 +1,4 @@
 Significance: patch
```

Comment:
I think the changelog filename looks a bit wonky (has spaces in) and also refers to a previous PR. Shall we delete it and create a fresh one with `jetpack changelog add`?

Reply:
huh.. good spotting. I'm not sure my rebase went quite as smoothly as I'd initially thought 😅 fixing it up now!